### DISCUSSION TITLE
New installation on Ubuntu

### DISCUSSION THREAD

**Discussion by 6-ril** [kimai/kimai#2656] [On-Premise: Installation & Updates] [answered]
Hello,
I'm installing Kimai, by following this doc : https://www.kimai.org/documentation/fresh-ubuntu-20.html

But when I run the command : 
bin/console kimai:install -n

I have an error in red : 
[ERROR] Failed to create database: An exception occurred in driver: SQLSTATE[HY000] [1045] Access denied for user `kiamai2`@`localhost`(using password: YES)

I create the user this way (like said in the doc)
CREATE USER IF NOT EXISTS `kimai2`@127.0.0.1 IDENTIFIED BY "my-super-secret-password";
GRANT select,insert,update,delete,create,alter,drop,index,references ON `kimai2`.* TO kimai2@127.0.0.1;

and this way too : 
CREATE USER IF NOT EXISTS `kimai2`@locahost IDENTIFIED BY "my-super-secret-password";
GRANT select,insert,update,delete,create,alter,drop,index,references ON `kimai2`.* TO kimai2@localhost;

Any idea of what's wrong ?

6Ril

**Comment by 6-ril** [comment]
SHOW DATABASES;
give me : 
information_schema
kimai2
mysql
performance_schema

SHOW GRANTS FOR `kimai2`@localhost
give me : 
GRANT USAGE ON *.* TO `kimai2`@`localhost` IDENTIFIED BY PASSWORD ' .... '
GRANT SELECT, INSERT, UPDATE, DELETE, CREATE, DROP, REFERENCES, INDEX, ALTER ON `kimai2`.* TO `kimai2`@`localhost`

**Comment by kevinpapst** [comment, marked as answer]
> Access denied for user `kiamai2@localhost`

Let me highlight that: Access denied for user ki**a**mai2@localhost

**Comment by 6-ril** [comment]
My mistake,
I'm using VirtualBox and cannot copy/paste directly in the VM, so I made a typo while copying.
I'm now using SSH, it's so much easier.


I was replying to you with the full list of command I used, then I figured out that it was just a typo in the password in the file .env 

Thanks for your answer, it helps me check everything one more time.

Cheers, 
Cyril
